src/main/java/net/sf/rails/ui/swing/StatusWindow.java

```diff
@@ -386,6 +386,26 @@ public void componentResized(ComponentEvent e) {
             }
         });
 
+        if ( Desktop.isDesktopSupported() && Desktop.getDesktop().isSupported(Desktop.Action.APP_QUIT_HANDLER) ) {
+            Desktop.getDesktop().setQuitHandler((e, r) -> {
+                if ( JOptionPane.showConfirmDialog(frame, LocalText.getText("CLOSE_WINDOW"), LocalText.getText("Select"), JOptionPane.OK_CANCEL_OPTION) == JOptionPane.OK_OPTION ) {
+                    frame.dispose();
+                    guiMgr.terminate();
+                    r.performQuit();
```

Comment:
What is `r`? Can you please give it a more meaningful name?^^

Reply:
:+1: